tests/unittests/cluster/test_ray_cluster.py

```diff
@@ -0,0 +1,23 @@
+import os
+import tempfile
+import time
+
+from autogluon.cloud.cluster import RayAWSClusterConfigGenerator, RayAWSClusterManager
+
+
+def test_ray_aws_cluster(test_helper):
```

Comment:
wondering what we are testing here. Shall we verify no error thrown for `up()` and `setup_connection()` or we should really test the connection is working

Reply:
Yea. We mainly test the `up()` and `setup_connection()` doesn't throw error now. Once the `RayJob` interface is added, we'll test the connection is actually working by submitting a job.